cac-prod.yml

```diff
@@ -13,7 +13,7 @@ jenkins:
         name: all
     - buildMonitor:
         includeRegex: >-
-          ^HMCTS_.*Platform.*\/(draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template)\/master
+          ^HMCTS_.*(Platform|CNP).*\/(draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template|cnp-rhubarb-recipes-service)\/master
```

Comment:
rhubarb is in the `CNP` folder

Reply:
did you want to do Nightly_CNP or are you trying to include the master build as well as nightly?